cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -594,3 +625,27 @@ func (c *CVCController) resizeCV(cv *apis.CStorVolume, newCapacity resource.Quan
 	}
 	return nil
 }
+
+// deletePDBIfNotInUse deletes the PDB if no volume is refering to the
+// cStorvolumeclaim PDB
+func (c *CVCController) deletePDBIfNotInUse(cvc *apis.CStorVolumeClaim) error {
+	pdbName := getPDBName(cvc)
+	cvcLabelSelector := string(apis.PodDisruptionBudgetKey) + "=" + pdbName
+	cvcList, err := c.clientset.
+		OpenebsV1alpha1().
+		CStorVolumeClaims(cvc.Namespace).
```

Comment:
can there be CVCs in different namespace using same PDB?

Reply:
No!! With current design all the OpenEBS related resources will be in one namespace where OpenEBS installed.